puppet/modules/slave/manifests/unittests.pp

```diff
@@ -74,7 +74,7 @@
   # nodejs/npm for JavaScript tests
   if $facts['os']['family'] == 'RedHat' {
     class { 'nodejs':
-      repo_url_suffix       => '12.x',
+      repo_url_suffix       => '14.x',
```

Comment:
This does imply our 3.7 & 3.8 PRs are now also testing with Node 14. I think that's OK, but IMHO not the worst reason to move more of our testing to GHA.

Reply:
Yes it does, and I agree that's OK for now.